build_tools/generate_dataops_stub.py

```diff
@@ -1,6 +1,6 @@
 # ruff: noqa
 
-# This script generates a .pyi (stub) file for skrub/_expressions/_expressions.py.
+# This script generates a .pyi (stub) file for skrub/_dataops/_dataops.py.
```

Comment:
Nitpick that applies to many places in this PR: the lower case of "DataOps" should be "data_ops".

We should be very systematic to always use underscore to separate words, to achieve consistency.

Sorry to be asking for a change like this, hopefully search and replace will do the trick